srml/randomness/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "srml-randomness"
+version = "1.0.0"
```

```suggestion
version = "2.0.0"
```

(not sure but we bumped all up after substrate 0.1 release)